html/Mogo/js/accordion/accordion.js

```diff
@@ -1,36 +1,49 @@
 import {components} from './components.js';
-import {hideEveryPanel, hidePanel, showPanel} from './helper-functions.js';
+import {
+  hideEveryPanel,
+  showPanel,
+  activeAccordion,
+  deactivateAccordions,
+  togglePannel,
+} from './helper-functions.js';
 
 export function addAccordionEffect() {
-  components.then((response) => {
-    const {accordion, panelElements, imgElement} = response;
+  components.then(
+      ({
+        accordionsList,
+        panelsList,
+        imgElement,
+        accordionsContainer,
+        images,
+      }) => {
+        activeAccordion(accordionsList[0]);
+        showPanel(panelsList[0]);
 
-    let visibleAccordionId = null;
-    let currentClickedAccorionId = null;
+        accordionsContainer.addEventListener('click', (event) => {
+          let target = event.target;
+          if (
+            target.classList.contains('services__accordion') ||
+          target.classList.contains('accordion__img') ||
+          target.classList.contains('accordion__arrow')
```

Comment:
as we were speaking, you could use here closest method here, it will simplify code a bit